keyserver/server.go

```diff
@@ -609,6 +625,28 @@ func (ks *Keyserver) step(step *proto.KeyserverStep, rs *proto.ReplicaState, wb
 	return
 }
 
+func (ks *Keyserver) takeOutOfRotation() {
+	ch := make(chan os.Signal, 1)
+	signal.Notify(ch, syscall.SIGUSR1)
+	for {
```

Comment:
Oh, you could also do the more idiomatic `for _ = range ch { ... }' (and then it would also close the loop if ever the channel was closed).